magpie/magpieadapter/__init__.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+import sys
+
+# TODO Is the following block really useful?
+this_dir = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(0, this_dir)
+
+
+from twitcher.adapter.base import AdapterInterface
+from twitcher.owsproxy import owsproxy
+
+
+logger = logging.getLogger(__name__)
+
+
+__version__ = '0.1.1'
```

Comment:
maybe return this in the API's `/version` route

Reply:
Forgot to remove that... don't thing that the adapter needs its own version